src/style.css

```diff
@@ -0,0 +1,63 @@
+/* stylelint-disable no-missing-end-of-source-newline */
```

Comment:
- [ ] It is not a good practice to disable any linter rule because you are creating room for future occurrences of bugs due to syntax errors. It is advisable to work towards solving the errors flagged by linters instead of disabling the rules. so please remove this comment which disable stylelint rule and make sure linters are passing 😃 👍